src/compas/artists/artist.py

```diff
@@ -106,18 +110,28 @@ def build_as(item, artist_type, **kwargs):
 
     @staticmethod
     def clear():
+        # deprecated: use `clear_scene` instead
```

Comment:
this deprecation is odd, because all of the newly written docs above use `clear()` instead of `clear_scene()`. Same thing goes for `draw`

Reply:
yes, fair enough, but then the docs need to be updated, because the method names don't make sense in my opinion :)